chapters/hub/hub-builder.md

```diff
@@ -0,0 +1,36 @@
+# Jina Hub Builder
+
+Jina’s hub-builder is a simple interface for building & validating Jina Hub executors. It is built on top of jina hub interface. It can be used as a Github action in the CICD workflow, or via CLI.
```

Comment:
- executors -> Executors
- CICD -> CI/CD
- Github -> GitHub
- `It is built on top of jina hub interface` - is `jina hub` the command or the project here? If command, wrap in backticks. If project, capitalize (please see @FionnD's style guide)